cdm/misc/src/test/java/ucar/nc2/geotiff/TestIFDEntry.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 1998-2024 University Corporation for Atmospheric Research/Unidata
+ * See LICENSE for license information.
+ */
+
+package ucar.nc2.geotiff;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.io.*;
+import java.lang.ReflectiveOperationException;
+import java.lang.invoke.MethodHandles;
+import java.lang.reflect.Method;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * IFDEntry read/write
+ *
+ * @author Ben Root
+ * @since 5/6/2024
+ */
+@RunWith(Parameterized.class)
+public class TestIFDEntry {
+  private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+  public IFDEntry ifd;
+  // Because we are dealing with reflection
+  public int testValue;
```

Comment:
I think these can be `private final`
```suggestion
  private final IFDEntry ifd;
  private final int testValue;
```